editor/components/block-list/with-hover-areas.js

```diff
@@ -57,6 +57,6 @@ const withHoverAreas = ( WrappedComponent ) => {
 	}
 
 	return WithHoverAreasComponent;
-};
+} );
```

Comment:
`createHigherOrderComponent` expects a second `name` argument. Though it's technically optional, helps when debugging through React DevTools.